Research/readme.md

```diff
@@ -1,9 +1,39 @@
 ﻿QuantConnect Research Project
 =============
+Currently we have two ways to use QuantConnect research notebooks, you can either install and run locally or just use our docker image (Recommended).
 
-There is an up to date docker image available at [quantconnect/research](https://hub.docker.com/repository/docker/quantconnect/research). You can pull this image with `docker pull quantconnect/research`.
+The up to date docker image available at [quantconnect/research](https://hub.docker.com/repository/docker/quantconnect/research). You can pull this image with `docker pull quantconnect/research`.
 
 
+# Running Jupyter Locally
```

Comment:
nit - I would move running locally after running on the docker image since we prefer users using that approach